Generals/Code/GameEngine/Include/GameClient/GameWindow.h

```diff
@@ -240,6 +241,9 @@ friend class GameWindowManager;
 	/// draw border for this window only, NO child windows or anything
 	virtual void winDrawBorder( void ) = 0;
 
+	void addTransitionWindow( TransitionWindow* transitionWindow );
+	void removeTransitionWindow( TransitionWindow* transitionWindow );
```

Comment:
Perhaps call these `linkTransitionWindow`, `unlinkTransitionWindow`. Because we are not really adding anything to the GameWindow. It is not doing anything with it. It just keeps this link.

Then the other function can be called `unlinkAllTransitionWindows` accordingly.